test/testground/network/sync.go

```diff
@@ -195,10 +193,7 @@ func DownloadGenesis(ctx context.Context, initCtx *run.InitContext) (json.RawMes
 	return genesis, nil
 }
 
-// DownloadSync downloads the given topic from the sync service. It will
-// download the given number of messages from the topic. If the topic is closed
-// before the expected number of messages are received, an error is returned.
-func DownloadSync[T any](ctx context.Context, initCtx *run.InitContext, topic *sync.Topic, t T, count int) ([]T, error) {
+func DownloadSync[T any](ctx context.Context, initCtx *run.InitContext, topic *sync.Topic, _ T, count int) ([]T, error) {
```

Comment:
Was the removal of the go documentation intentional?

Reply:
no it wasn't good catch, I think was some issue with merging. fixed in 4bcb9d03d88e4bde3227feaff84a1fdae0b0c68c